hedera-node/src/main/java/com/hedera/services/txns/validation/ContextOptionValidator.java

```diff
@@ -166,7 +166,7 @@ public ResponseCodeEnum nftMaxQueryRangeCheck(int start, int end) {
 		return lengthCheck(
 				end - start,
 				dynamicProperties.maxNFTQueryRange(),
-				ResponseCodeEnum.QUERY_RANGE_LIMIT_EXCEEDED
+				ResponseCodeEnum.INVALID_QUERY_RANGE
```

Comment:
Lets not include this change in this PR. It is related to the `info` query you are working on